docs/cli/installing-the-pulse-cli.md

```diff
@@ -4,42 +4,42 @@ Download the latest version of the Pulse CLI for your operating system and make
 
 1.  Take note of the latest version of the CLI:
 
-    [![Download](https://api.bintray.com/packages/codacy/pulse/event-cli/images/download.svg)](https://bintray.com/codacy/pulse/event-cli/_latestVersion)
+    [![Download](https://img.shields.io/github/v/release/codacy/pulse-event-cli?sort=semver)](https://artifacts.codacy.com/codacy/pulse/event-cli/latest)
```

Comment:
When I open https://artifacts.codacy.com/codacy/pulse/event-cli/latest I get:

```bash
$ curl https://artifacts.codacy.com/codacy/pulse/event-cli/latest
1.9.0
```

Is it important to keep this link? Before it would point to a page with information about the package and links to download it. But it we don't have something more equivalent I suggest that we remove it:

```suggestion
    [Latest version](https://img.shields.io/github/v/release/codacy/pulse-event-cli?sort=semver)
```